### ISSUE TITLE
Multi streams in multiparty chatroom

### ISSUE THREAD

**Issue by davidroettger** [open-easyrtc/open-easyrtc#112]
In the multiparty chatroom demo the setRoomOccupantListener is set to null. 
I would use this listener to listen for new users and than add the stream to the call. Is there another way to tell a new user in the room about the existing streams?

**Comment by hthetiot** [comment]
Yes, look at this sample, it create a button to perform call, but you can perform call, but you can perform call if user is the most recent, carefull not to double call same users pair.

```js
easyrtc.setRoomOccupantListener(convertListToButtons);
```

https://github.com/open-easyrtc/open-easyrtc/blob/b970d425b54da64140dcedfc02a5117952451d82/demos/js/demo_lowbandwidth.js#L50
